routes/index.js

```diff
@@ -1,9 +1,11 @@
 const emails = require('./emails');
 const conversations = require('./conversations');
+const confirmation = require('./confirmation');
```

Comment:
confirmation shouldn't have its own routes - they should be under conversations.
`/conversations/:id/verify?token=VERIFICATION_TOKEN`